swagger.yaml

```diff
@@ -6491,6 +6496,12 @@ definitions:
         x-nullable: false
         default: 'x86_64'
         description: The CPU architecture of the image (x86_64/arm64/etc).
+      discovery_kernel_parameters:
+        description: List of string values to be passed to discovery image as kernel parameters.
```

Comment:
Sounds good, but not clear to me why we have two different descriptions for the same field?

1) `Json formatted string array representing the discovery image kernel parameters.`
2) `List of string values to be passed to discovery image as kernel parameters.`

Reply:
In the parameters to the API call we get them as array, while in the database it is stored as JSON formatted string